cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -718,13 +746,18 @@ def test_load_values_on_init_option_service_dialog_element():
         casecomponent: Services
         initialEstimate: 1/16h
         testSteps:
-            1. create a service dialog
+            1. create a service dialog with option dynamic to True
```

Comment:
It looks like part of test steps not setup. If yes, I would try to create dynamic service dialog as part of test only.
Second thought if it setup; we already have one fixture `service_dialog` better try to modify that in state of creating new fixture.

Reply:
dialog creation is part of testcase so updated tc accordingly.